DESIGN.md

```diff
@@ -0,0 +1,256 @@
+# Design: Redalert Image Testing
+## Author: Mathew Robinson <chasinglogic@gmail.com>
+
+## Summary
+Redalert is a tool used for system validation. Similar to ChefSpec/ServerSpec
+however written in Python and configured with YAML to be more familiar to those
+used to working with Ansible.
```

Comment:
Redalert is a tool used for system validation, similar to ChefSpec/ServerSpec.
It is written in Python and configured with YAML to be more familiar to those accustomed to working with Ansible.